electron/src/services/electronPlatformUtils.service.ts

```diff
@@ -192,8 +195,17 @@ export class ElectronPlatformUtilsService implements PlatformUtilsService {
         return ipcRenderer.invoke('systemTheme');
     }
 
-    onDefaultSystemThemeChange(callback: ((theme: 'light' | 'dark') => unknown)) {
-        ipcRenderer.on('systemThemeUpdated', (event, theme: 'light' | 'dark') => callback(theme));
+    onDefaultSystemThemeChange(callback: ((theme: ThemeType.Light | ThemeType.Dark) => unknown)) {
+        ipcRenderer.on('systemThemeUpdated', (event, theme: ThemeType.Light | ThemeType.Dark) => callback(theme));
+    }
+
+    async getEffectiveTheme() {
```

Comment:
Would be nice to replace some of the logic in `services.module` in the various projects with this new logic. (To avoid duplicated code)